src/containers/CheckoutPage/CheckoutPage.js

```diff
@@ -153,26 +159,27 @@ export class CheckoutPageComponent extends Component {
     const {
       onInitiateOrder,
       onHandleCardPayment,
-      // onConfirmPayment,
+      onConfirmPayment,
+      onSendInitialMessage,
     } = this.props;
 
     // Step 1: initiate order by requesting payment from Marketplace API
     const fnRequestPayment = fnParams => {
       // fnParams should be { listingId, bookingStart, bookingEnd }
-      const {
-        pageData,
-        stripePaymentIntentClientSecret,
-        stripePaymentIntentId,
-      } = handlePaymentParams;
+      const { pageData } = handlePaymentParams;
+
+      const { stripePaymentIntentClientSecret, stripePaymentIntentId } = fnParams;
```

Comment:
This is step 1, so these stripePaymentIntentClientSecret and stripePaymentIntentId are not actually available unless user has completed `onInitiateOrder` call previously. My previous code was expecting clientSecret to be saved in sessionStorage, but it will likely be saved among transaction entity.

Just remove this line and in line 175 return `? Promise.resolve(pageData.enquiredTransaction)`. 
(Although, I'll change that line in my PR.)